lib/looker.ts

```diff
@@ -0,0 +1,70 @@
+import { LookerNodeSDK } from "@looker/sdk-node"
+import { IDashboardElement, IWriteQuery } from "@looker/sdk"
+
+/**
+ *
+ * @type {string} Local configuration file name, one directory above
+ */
+
+console.log('LOOKER ENABLED: ', process.env.IS_LOOKER_ENABLED);
+
+export async function getAWDashboardElement0(SDK: any): Promise<IDashboardElement> {
+  const dashboardId = "1471";
+//  await(getDBFilters())
+
+  const elements: IDashboardElement[] = await SDK.ok(SDK.dashboard_dashboard_elements(dashboardId));
+  return elements[0];
+}
+
+export async function runEventCountQuery(filters: any): Promise<any>{
+  const SDK = LookerNodeSDK.init40()
+  const me = await SDK.ok(SDK.me());
```

Comment:
Your concern is valid. How about moving the `LookerNodeSDK.init40()` line and assignment to the top-level, where it will be executed and module load time, and getting rid of the `SDK.me` line entirely, since we don't use the result...

Reply:
When removing `SDK.me` and moving the `LookerNodeSDK.init40()` line to the top-level, there are still build failures that show this error:
```
Error: TS-SDK configuration error:
Missing required configuration values like base_url
```
If we are trying to use the service account in prod, does it make sense to add the `IS_LOOKER_ENABLED` and `LOOKERSDK_BASE_URL` env variables to the `.env` file in order to avoid these build failures?

Reply:
This is definitely the right question.  Generally, this doesn't seem like a great idea, but I'm wondering if the is going to end up being yes in this case, since we want to pre-render the web pages.  I'm reading through some server components and poking around a bit as I think about this.

Reply:
So I spent a bunch of time this afternoon wrapping my head around the many possible ways to structure NextJS 14 static vs dynamic pages and how that interleaves with caching, and I will start by saying that I'm very glad the NextJS 15 docs talk a bunch about how they're going to keep working to make this better.  The complexity of the developer experience is currently.... painful.

In any case, my biggest conclusion is that we don't want to block landing this PR on getting all the caching stuff sorted out. 

This makes me think that my suggestion of hoisting the SDK initialization isn't the right way to go.  Since the init40 method isn't obviously idempotent, I'd suggest we move the SDK initialization stuff to a method in looker.ts that looks something like this:

```
let SDK;

function getLookerSDK() {
	if (!SDK) {
		SDK = LookerNodeSDK.init40()
	}
	return SDK
}
```

More Tuesday, though I'm not quite sure when, I have some meetings and need to prioritize Road Show work over the CTR stuff, unforch.

Reply:
Ahh I see. I'll move the SDK initialization as per your suggestion for now and hopefully we can find some time to pair soon!